src/Extensions/MemberLockoutExtension.php

```diff
@@ -57,29 +58,50 @@ public function getLockIfInactiveAfter() : DBDateTime
         $defaultLockoutThreshold = Config::inst()->get(Member::class, 'lockout_threshold_days');
 
         $groups = $this->owner->Groups();
-        $lowestThreshold = $groups->filter([
+        $lowestThreshold = $groups->filter(
+            [
             'LockoutThresholdDays:GreaterThan' => 0,
             'LockoutThresholdDays:LessThan' => $defaultLockoutThreshold,
-        ])->min('LockoutThresholdDays') ?: $defaultLockoutThreshold;
+            ]
+        )->min('LockoutThresholdDays') ?: $defaultLockoutThreshold;
 
         return DBDateTime::now()->modify("-{$lowestThreshold} days");
     }
 
     /**
      * Return the date the user has last accessed the CMS
      */
-    public function getLastAccessed() : DBDatetime
+    public function getLastAccessed()
```

Comment:
No need to remove the type
```suggestion
    public function getLastAccessed(): ?DBDateTime
```